### PR TITLE
[IMP] l10n_no: update COA data

### PR DESCRIPTION
**Pull request by yosuanicolaus**
Update the norwegian COA according to the official COA https://github.com/Skatteetaten/saf-t/blob/master/General%20Ledger%20Standard%20Accounts/CSV/General_Ledger_Standard_Accounts_4_character.csv

task-id: 2716886
enterprise-PR: https://github.com/odoo/enterprise/pull/46547

### CODE REVIEW

**Review by malb-odoo — COMMENTED** [review]
Small details to fix when you have the time :smile:

File: addons/l10n_no/data/template/account.account-no.csv [hunk 1]
```diff
@@ -1,473 +1,746 @@
...
+"chart3260","3260","Withdrawal of goods, exempt from VAT","income","","False","Uttak av varer, unntatt avgiftsplikt"
+"chart3281","3281","Cash discounts, exempt from VAT","income","","False","Kontantrabatt, unntatt avgiftsplikt"
+"chart3285","3285","Quantity discounts, exempt from VAT","income","","False","Kvantumsrabatt, unntatt avgiftsplikt"
+"chart3289","3289","Other reductions in sales revenue, exempt from VAT","income","","False","Annen salgsinntektsreduksjon, unntatt avgiftsplikt"
+"chart3290","3290","Re-invoicing of sales costs, exempt from VAT","income","","False","Viderefakturering av omkostninger ved salg, unntatt avgiftsplikt"
+"chart3300","3300","Special government fees and taxes on goods manufactured/sold","income","","False","Spesiell offentlig avgift for tilvirkede/solgte varer"
+"chart3400","3400","Special public grant for manufactured/sold goods","income","","False","Spesielt offentlig tilskudd for tilvirkede/solgte varer"
+"chart3440","3440","Special public grant for services","income","","False","Spesielt offentlig tilskudd for tjeneste"
+"chart3500","3500","Guarantees","income","","False","Garanti"
+"chart3510","3510","Service","income","","False","Service"
```
[the remark this answers is not in the document]
Comment (yosuanicolaus) [reply addons/l10n_no/data/template/account.account-no.csv:389]: From quick google translate it seems that "Service" in Norwegian is also "Service" :)

[the remark this answers is not in the document]
Comment (malb-odoo) [reply addons/l10n_no/data/template/account.account-no.csv:389]: I have some alternative on deepl but let it be Service then 
image

**Review by malb-odoo — APPROVED** [review]
LGTM good job !

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**